Source/Scene/Points3DTileContent.js

```diff
@@ -148,19 +155,42 @@ define([
         }
     });
 
+    function createFeatures(content) {
```

Comment:
Basically it's the difference between `PointsBatched` and `PointsWithPerPointProperties`.